### ISSUE TITLE
Dashboard accessibility

### ISSUE THREAD

**Issue by Abumpus1** [Abumpus1/overlook#14]
Accessibility audit on dashboard should be 100%
Should be able to use app without trackpad
ensure clarity for screenreaders (ARIA attributes where needed)
